addon/-private/system/relationships/state/create.js

```diff
@@ -6,8 +6,13 @@ import EmptyObject from "ember-data/-private/system/empty-object";
 var get = Ember.get;
 
 function createRelationshipFor(record, relationshipMeta, store) {
-  var inverseKey;
-  var inverse = record.type.inverseFor(relationshipMeta.key, store);
+  let inverseKey;
+  let inverse = null;
+  if (!relationshipMeta.options ||
```

Comment:
Gotcha. Is `false` also a documented option for inverse? I haven't seen it before.